packages/scm/src/browser/scm-widget.tsx

```diff
@@ -36,11 +36,11 @@ export class ScmWidget extends BaseWidget implements StatefulWidget {
 
     static ID = 'scm-view';
 
-    @inject(ScmService) protected readonly scmService: ScmService;
+    @inject(ScmService) readonly scmService: ScmService;
```

Comment:
@akosyakov is the idea that we want to expose these injections instead of accessing them internally?
My worry is it will give too much access to others, but you mentioned that if we need them during the test, downstream apps might find it useful as well.

> If you need them to write tests, it means that downstream will find them useful too.

Reply:
for this one you can use `container.get(ScmService)`, others are fine